Yoast/wordpress-seo
# Upgraded react and react-dom to 16.8.6

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Upgraded react to 16.8.6.

## Relevant technical choices:

* Chose to upgrade to `16.8.6` because it is the same version as `@wordpress/element`.

## Test instructions

This PR can be tested by following these steps:

For developers:
* Run `yarn why react-dom` and make sure react-dom `16.2.0` is no longer used.

General:
* Make sure all reactified elements in the plugin still work in WordPress `4.9.8`.

## UI changes

## Quality assurance


Fixes #12471
